docs/content/specs-defs/specs/shared/_index.md

```diff
@@ -119,15 +118,15 @@ These will also be provided as a comment on the module proposal, once accepted,
 
 {{< /hint >}}
 
-Modules **MUST** provide the capability to collect deployment/usage telemetry, via a blank ARM deployment, as detailed in [Telemetry](/Azure-Verified-Modules/help-support/telemetry/) further. Telemetry data should be collected on the "top level" `main.bicep` or `main.telemetry.tf` file; it is not necessary to include it in any nested files (child modules).
-
+Modules **MUST** provide the capability to collect deployment/usage telemetry, via a blank ARM deployment, as detailed in [Telemetry](/Azure-Verified-Modules/help-support/telemetry/) further. Telemetry data should be collected on the "top level" `main.bicep` or `main.telemetry.tf` file.
```

Comment:
This is not longer correct? It is not a blank ARM deployment for Terraform. Do we need to break up the wording here?

Reply:
Updated this and tlinked file

Reply:
@jaredfholgate cna you take another look please